src/webcomponents/commons/filters/hpo-accessions-filter.js

```diff
@@ -47,7 +45,7 @@ export default class HpoAccessionsFilter extends LitElement {
         this._selectedTermsArr = [];
         this.ontologyTerm = "HPO";
         this.ontologyFilter = "hp";
-        this._config = {...this.getDefaultConfig(), ...this.config};
+        this._config = {...this.getDefaultConfig()};
```

Comment:
The `this.getDefaultConfig` is returning a new object, so using the `{... }` in this case is redundant. 

You can just use `this._config = this.getDefaultConfig();` :wink:.